controller/QtiCreator.php

```diff
@@ -291,6 +291,12 @@ protected function getCreatorConfig(core_kernel_classes_Resource $item)
         $lang = \common_session_SessionManager::getSession()->getDataLanguage();
         $config->setProperty('lang', $lang);
 
+        // TODO: better implement the following parameters
+        // --- start
+        $config->setProperty('translation', $this->getRequestParameter('translation'));
+        $config->setProperty('origin', $this->getRequestParameter('originResourceUri'));
```

Comment:
Sounds good to me, thanks. My personal preference, which is not a must here is to use the `originResourceUri` instead of `origin`, to make it more explicit